src/components/stack-chart/Canvas.tsx

```diff
@@ -624,6 +625,17 @@ class StackChartCanvasImpl extends React.PureComponent<Props> {
       timelineUnit === 'bytes'
         ? formatBytes(duration)
         : formatMilliseconds(duration);
+    let argumentSummaries = undefined;
+    if (timing.argumentValues) {
+      const argumentValues = timing.argumentValues[stackTimingIndex];
+      if (argumentValues !== -1) {
+        argumentSummaries = ValueSummaryReader.getArgumentSummaries(
```

Comment:
Hmm typescript says the return value of `ValueSummaryReader.getArgumentSummaries` is `any` which isn't very helpful. Do you know what it returns? Ideally it would be good to type them.